### DISCUSSION TITLE
I can't Finish initial setup for my git repository URL manually for gitea

### DISCUSSION THREAD

**Discussion by JonathanAJ** [codemagic-ci-cd/codemagic-docs#2496] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Im using Gitea and selected the option "Add URL manually" on codemagic.

![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/12076722/6b8dbd0e-3bb1-4161-899e-312071140310)

On second step I enter my repository credentials and the codemagic show success for my configuration.

But I can't click on next step to finish my configuration.

![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/12076722/bacecbcf-3d2e-437e-97e0-9a9812c72d31)

The finish step dont work for me.

### Expected results

The finish step work and i can configure my repository

### Actual results

The finish step don't work and i can't configure my repository

**Comment by dtrdic** [comment]
https://discord.com/channels/1131597315707261018/1233500058498895882/1233500058498895882
